### PR TITLE
Fix help for `show`

### PR DESCRIPTION
**Pull request by PawelLipski**

### CODE REVIEW

File: git_machete/cmd.py [hunk 1]
```diff
@@ -2281,8 +2281,9 @@ def strip_first_fragment(rb: str) -> str:
             cb = current_branch(git)
             rebase_onto_ancestor_commit(git, cb, cli_opts.opt_fork_point or machete_client.fork_point(cb, use_overrides=True))
         elif cmd == "show":
-            param = check_required_param(args[:1], allowed_directions(allow_current=True))
-            branch = check_optional_param(args[1:])
+            args1 = parse_options(args)
```
Comment (PawelLipski) [inline git_machete/cmd.py:2287]: Whoops

File: git_machete/docs.py [hunk 2]
```diff
@@ -414,18 +422,18 @@
     "show": """
         <b>Usage: git machete show <direction> [<branch>]</b>
         where <direction> is one of: `c[urrent]`, `d[own]`, `f[irst]`, `l[ast]`, `n[ext]`, `p[rev]`, `r[oot]`, `u[p]`
-        displayed relative to target <branch>, or the current checked out branch if <branch> is unspecified.
+        displayed relative to given <branch>, or the current checked out branch if <branch> is unspecified.
```
Comment (PawelLipski) [inline git_machete/docs.py:425]: To remove the confusion that @Sarunoi pointed out today

File: git_machete/cmd.py (hunk 1 above)
Comment (rifel123) [reply git_machete/cmd.py:2287]: Hmm, is it necessary to call parse_options again? All in all it's called before   at 2122
`cmd_and_args = parse_options(orig_args, gnu=False)`

Comment (PawelLipski) [reply git_machete/cmd.py:2287]: It is necessary... let me explain in the code why; anyway, I hope this will all be gone with the advent of #158

**Review by rifel123 — APPROVED** [review]

**Pull request merged**